claat/parser/md/parse.go

```diff
@@ -365,10 +365,12 @@ func addMetadataToCodelab(m map[string]string, c *types.Codelab) error {
 		case metaCategories:
 			// Standardize the categories and append to codelab field.
 			c.Categories = append(c.Categories, standardSplit(v)...)
+			sort.Strings(c.Categories)
 			break
 		case metaEnvironments:
 			// Standardize the tags and append to the codelab field.
 			c.Tags = append(c.Tags, standardSplit(v)...)
+			sort.Strings(c.Tags)
```

Comment:
Isn't sorting already done in `func parseMarkup` at the very end?
https://github.com/googlecodelabs/tools/blob/c0f3afc2f3a7436818b5163d1465f6c07e33e32d/claat/parser/md/parse.go#L213

Reply:
Ah you're right.  What I thought was unstable ordering was actually not!